cli/resource.go

```diff
@@ -72,3 +77,28 @@ func runResourceAdd(args *docopt.Args, client *controller.Client) error {
 
 	return nil
 }
+
+func runResourceRemove(args *docopt.Args, client *controller.Client) error {
+	provider := args.String["<provider>"]
+	resource := args.String["<resource>"]
+
+	res, err := client.DeleteResource(provider, resource)
+	if err != nil {
+		return err
+	}
+
+	// Unset all the keys associated with this resource
+	env := make(map[string]*string)
+	for k := range res.Env {
+		env[k] = nil
```

Comment:
Yeah, I am not sure.
There is somewhat of an assumption variables introduced by the resource are associated with it and if they were to be modified it should have been done to modify the behaviour of the application specific to that resource.
That said is there a means of comparing it? The original value of the variables is only known at resource provisioning time (I think).